source/common/upstream/load_balancer_impl.h

```diff
@@ -88,14 +101,22 @@ class LoadBalancerBase : public LoadBalancer {
   // Calculating normalized total health starts with summarizing all priorities' health values.
   // It can exceed 100%. For example if there are three priorities and each is 100% healthy, the
   // total of all priorities is 300%. Normalized total health is then capped at 100%.
-  static uint32_t calcNormalizedTotalHealth(std::vector<uint32_t>& per_priority_health) {
-    return std::min<uint32_t>(
-        std::accumulate(per_priority_health.begin(), per_priority_health.end(), 0), 100);
+  static uint32_t calcNormalizedTotalHealth(std::vector<uint32_t>& per_priority_health,
```

Comment:
Can you update the comment above to talk about what this does now with regard to normalizing healthy + degraded? It's not immediately clear to me. I guess for the purpose of computing panic mode, etc. we consider healthy + degraded together? Maybe make that a bit more clear in the comments?